index.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+// const fs = require('fs');
+// const expect = require('chai').expect;
+const readfiles = require('./lib/read-files.js');
+const main = module.exports = function (argv){
+    if(argv.length !=5) return 'Please enter at least 3 arguments';
+        return readfiles(argv.slice(2,5), function(err, data){
+            if (err) {
+                return console.error(err);
+            }
+            return console.log(data);
+    })
+}
+
+console.log(main(process.argv));
+// let bacon = `${__dirname}/assets/bacon.txt`;
```

Comment:
Sorry for the "unnecessary-ness" of this because I overlooked the instructions on using this int he command line, but I hope it helped mildly.